05-regression.Rmd

```diff
@@ -153,11 +153,11 @@ You can do this by using RStudio's spreadsheet viewer or by using the `glimpse()
 glimpse(evals_ch5)
 ```
 
-Observe that `Observations: 463` indicates that there are 463 rows/observations in `evals_ch5`, where each row corresponds to one observed course at UT Austin. It is important to note that the *observational unit* \index{observational unit} is an individual course and not an individual instructor. Recall from Subsection \@ref(exploredataframes) that the observational unit is the "type of thing" that is being measured by our variables. Since instructors teach more than one course in an academic year, the same instructor will appear more than once in the data. Hence there are fewer than 463 unique instructors being represented in `evals_ch5`. We'll revisit this idea in Section \@ref(regression-conditions), when we talk about the "independence assumption" for inference for regression.
+Observe that ``Observations: `r nrow(evals_ch5)` `` indicates that there are `r nrow(evals_ch5)` rows/observations in `evals_ch5`, where each row corresponds to one observed course at UT Austin. It is important to note that the *observational unit* \index{observational unit} is an individual course and not an individual instructor. Recall from Subsection \@ref(exploredataframes) that the observational unit is the "type of thing" that is being measured by our variables. Since instructors teach more than one course in an academic year, the same instructor will appear more than once in the data. Hence there are fewer than `r nrow(evals_ch5)` unique instructors being represented in `evals_ch5`. We'll revisit this idea in Section \@ref(regression-conditions), when we talk about the "independence assumption" for inference for regression.
```

Comment:
Same comment regarding dimensions of `evals`. Let's compute this value only once right after `evals` is loaded and save in an interval variable called `n_courses_ch5`